build.gradle

```diff
@@ -158,9 +158,9 @@ subprojects {
                 protobuf_plugin: 'com.google.protobuf:protobuf-gradle-plugin:0.8.0',
                 protobuf_util: "com.google.protobuf:protobuf-java-util:${protobufVersion}",
 
-                netty: 'io.netty:netty-codec-http2:[4.1.4.Final]',
+                netty: 'io.netty:netty-codec-http2:[4.1.6.Final-SNAPSHOT]',
                 netty_epoll: 'io.netty:netty-transport-native-epoll:4.1.4.Final' + epoll_suffix,
-                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork19',
+                netty_tcnative: 'io.netty:netty-tcnative-boringssl-static:1.1.33.Fork22',
```

Comment:
Sorry if I wasn't clear. I ll remove the changes in the build.gradle file before merge. Just left them in there for now, so I can build locally.